tests/math_builtin_api/CMakeLists.txt

```diff
@@ -1,9 +1,17 @@
 set(TEST_CASES_LIST "")
 
 set(MATH_CAT_WITH_VARIANT common float relational)
-set(MATH_VARIANT base double half)
+set(MATH_VARIANT base)
 
-set(MATH_CAT integer geometric native half)
+set(MATH_CAT integer geometric native)
```

Comment:
Just want to point out that half in MATH_CAT is for Half precision math functions (Table 173) and they operate on float. So maybe they shouldn't be excluded even if half tests are not enabled.

Reply:
Good point, fixed